### DISCUSSION TITLE
VGA was probably working all along

### DISCUSSION THREAD

**Discussion by ej012345** [picocomputer/community#12] [Q&A]
I can be very slow... after several days of wiring verification and 2 fixes, still no VGA :( 
But I had a thought, my Dell monitor has a VGA port, maybe I have a hardware issue, I should try a direct VGA cable. Eureka!
Went to bed on that, but had a nagging thought... uh, didn't VGA to HDMI converter come with a USB power cable?!?!
Oh for criminy sakes!  That made it work too, sometimes, for now I'll just stick with the VGA cable.
**DosPicos6502 lives!**
![p-23 06 09-01](https://github.com/picocomputer/discussions/assets/131208326/8dccb81a-c490-4318-8886-3fb391d2c9c4)
![p-23 06 09-08](https://github.com/picocomputer/discussions/assets/131208326/1fa56f8d-6a26-43bb-a7fc-4537cde1bdbd)

Hello, World!
![p-23 06 09-12](https://github.com/picocomputer/discussions/assets/131208326/f97be165-f9fb-4e34-ba85-ef0d27a8962a)

**Comment by ej012345** [comment]
P.S. I love the new schematic! very easy to follow.

**Comment by rumbledethumps** [comment]
Make sure you swap PHI2-PWMR as shown on the new schematic before loading the latest firmware. Your VGA-HDMI gizmo can power itself from pin 9 of the VGA port, which is also shown on the new schematic. There will be no more schematic changes since I have working circuit boards now.

**Comment by ej012345** [comment]
Cool… that wasn’t bad
![IMG_2608](https://github.com/picocomputer/discussions/assets/131208326/bd9ed264-3f99-4b41-bf32-33447538936c)

**Reply by MatthewRohrich** [reply]
It took me a little while to work out that the VGA to HDMI adapter I have is missing the wire for pin 9. So the USB cable solved it for me too.
